src/com/duckduckgo/mobile/android/adapters/RecyclerMainFeedAdapter.java

```diff
@@ -103,9 +120,7 @@ public HeaderViewHolder(View itemView) {
             instructionbutton = (Button) itemView.findViewById(R.id.instruction_button);
             dismissImageButton = (ImageButton) itemView.findViewById(R.id.dismiss_image_button);
             viewPager = (ViewPager) itemView.findViewById(R.id.view_pager);
-            adapter = new BannerOnboardingAdapter(fragmentManager);
-            viewPager.setAdapter(adapter);
-            viewPager.setPageTransformer(false, new OnboardingTransformer());
+            viewPager.setAdapter(new BannerOnboardingAdapter());
             viewPager.addOnPageChangeListener(new ViewPager.OnPageChangeListener() {
```

Comment:
is this minSdk 9? its from the v4 package, and I think we are still using 23 from the support libraries, wasn't the minSdk bumped to 9 from the 25.version?
I agree that we should bump the target/compile sdk and support libraries anyways, but I'll make a different PR for this.
I don't know how many devices we have still on 8, assume not many, for the new app I agree we will have a higher minSdk!